boards/arm/96b_wistrio/pinmux.c

```diff
@@ -33,8 +43,38 @@ static int pinmux_stm32_init(struct device *port)
 
 	stm32_setup_pins(pinconf, ARRAY_SIZE(pinconf));
 
+	struct device *gpioa =
```

Comment:
Please introduce all the variables at the beginning of the function.

Reply:
Okay